fw/rbcx-coprocessor/src/Bsp.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Bsp.hpp"
+#include "Esp32Manager.hpp"
+
+extern "C" void EXTI9_5_IRQHandler(void) {
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_5);
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_6);
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_7);
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_8);
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_9);
```

Comment:
Nechcem mít maximum definované v *hpp* ? Když tady jako parametry použijem naše názvy z *hpp*, mohlo by to být čistější.

Reply:
Btw nevidím, kde povoluješ ty EXTI srandy (nemyslím NVIC). Fakt to sem lítá?

Reply:
Je to handler přerušení, tj. musí být nadefinován jen jednou (v includu by byl nadefinován v každém souboru, který to includuje).

O EXTI srandách nic nevím, ale lítá to tam. Myslím, že je povoluje HAL https://github.com/STMicroelectronics/STM32CubeF1/blob/441b2cbdc25aa50437a59c4bffe22b88e78942c9/Drivers/STM32F1xx_HAL_Driver/Src/stm32f1xx_hal_gpio.c#L288

Reply:
Jj už jsem to taky našel. Tamto myslím jen definovat piny v hpp, tady pak jen:

```cpp
    HAL_GPIO_EXTI_IRQHandler(espPinBla);
    HAL_GPIO_EXTI_IRQHandler(espPinBle);
```

Reply:
No, jako ten IRQ handler je společný pro všechny ty piny, co tam mám teď. Dal jsem je tam pro jistotu, aby když to bude někdo používat s dalšími piny se na to nezapomnělo a nebylo z toho mrzení.

Nepoužívam definice z BSP protože `EXTI9_5_IRQHandler` bude vždycky na tyhlety piny, to bychom museli vyměnit STM aby se to změnilo.

Reply:
Jako ono to nebude ještě tak jednoduchý, na novým HW bude *EXTI9_5* použitý pro ultrazvuky :)

Reply:
Už chápu